documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -1,54 +1,125 @@
 > Release Issue Template
 
-# Lotus X.Y.Z Release
-
-[//]: # (Open this issue as [WIP] Lotus vX.Y.Z)
-[//]: # (Apply the `tpm` label to it, and pin the issue on GitHub)
+# Lotus Node|Miner X.Y.Z Release
+
+[//]: # (Below are non-visible steps intended for the issue creator)
+[//]: # (Start an issue with title "[WIP] Lotus Node|Miner vX.Y.Z" and adjust the title for whether it's a Node or Miner release.)
+[//]: # (Copy in the content of https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md)
+[//]: # (Find/Replace "X.Y.Z" with the actual values.)
+[//]: # (Copy/paste the "Release Checklist > RCX" section to "Release Checklist > Stable \(non-RC\) Release" and apply the "diff" called out there.)
+[//]: # (Find/Replace "RCX" with "RC1".)
+[//]: # (If this isn't a release tied to a network upgrade, remove all items with "\(network upgrade\)")
+[//]: # (Adjust the "Meta" section values.)
+[//]: # (Apply the `tpm` label to the issue)
+[//]: # (Create the issue)
+[//]: # (Pin the issue on GitHub)
+
+## 😶‍🌫 Meta
+* Scope: Node|Miner
+* Is this linked with a netwok upgrade, and thus mandatory? Yes|No
+* Related network upgrade version: n/a|nvXX
 
 ## 🚢 Estimated shipping date
 
-<Date this release will ship on if everything goes to plan (week beginning...)>
-
-## ✅ Release Checklist
-
-**Note for whoever is owning the release:** please capture notes as comments in this issue for anything you noticed that could be improved for future releases.  There is a *Post Release* step below for incorporating changes back into the [RELEASE_ISSUE_TEMPLATE](https://github.com/filecoin-project/lotus/blob/master/documentation/misc/RELEASE_ISSUE_TEMPLATE.md), and this is easier done by collecting notes from along the way rather than just thinking about it at the end.
+[//]: # (If/when we know an exact date, remove the "week of".)
+[//]: # (If a date week is an estimate, annotate with "estimate".)
 
-- [ ] Fork a new branch (`release/vX.Y.Z` or `release/miner/vX.Y.Z`) from `master` and make any further release related changes to this branch. If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.
```

Comment:
> If any "non-trivial" changes get added to the release, uncheck all the checkboxes and return to this stage.

Instead of an "uncheck" flow, I am going with a "copy a new section" flow so it's clear what RC we're on and what has been done.

Reply:
I like this. Additionally it makes it clearer if release engineer needs to hand over the release duties in the middle of a RC-train